mix.exs

```diff
@@ -4,8 +4,8 @@ defmodule EnvVarProvider.MixProject do
   def project do
     [
       app: :env_var_provider,
-      version: "0.4.1",
-      elixir: "~> 1.7",
+      version: "0.4.2",
+      elixir: "~> 1.9",
```

Comment:
Not necessary but also not necessary to update otherwise, so I took the chance to just update this now.